e2e-ops.sh

```diff
@@ -11,15 +11,16 @@ set -eExuo pipefail
 
 export MCG_DISPLAY_NAME="MCG OSD Deployer"
 # Check if the addon is already successfully installed.
-if [[ $(oc get csv -n redhat-data-federation | grep mcg-osd-deployer | wc -l) -gt 0 ]]; then
-# @TODO: uncomment this check when the after-patch issue (operator transitioning from "Succeeded" to "Installing" for a long time) is resolved.
-#  timeout 10m bash <<-'EOF'
-#  until [[ $(oc get csv -n redhat-data-federation -o=jsonpath="{.items[?(@.spec.displayName==\"${MCG_DISPLAY_NAME}\")].status.phase}") == "Succeeded" ]]; do
-#      echo "Waiting for ${MCG_DISPLAY_NAME} to reach succeeded state..."
-#      sleep 5
-#  done
-#EOF
-  exit 0
+if [[ $(oc get csv -n redhat-data-federation | grep -c mcg-osd-deployer) -gt 0 ]]; then
+    timeout 3m bash <<-'EOF'
+    # We are in a subshell, export again.
+    export MCG_DISPLAY_NAME="MCG OSD Deployer"
```

Comment:
Is this extra export needed?  When running locally it is not...

Reply:
Yes, at least when running in plain local: `./e2e-ops.sh`

Reply:
Removed.